Libraries/Opc.Ua.Client/NodeCache/INodeCache.cs

```diff
@@ -135,5 +135,10 @@ ValueTask<string> GetDisplayTextAsync(
         NodeId BuildBrowsePath(
             ILocalNode node,
             IList<QualifiedName> browsePath);
+
+        /// <summary>
```

Comment:
I am not sure where we would need this, and it is a new thing that needs to be implemented again on INodeCache implementers.  Better add a extension method that takes a NodeCache and converts.  But again, even that is not used anywhere in this PR, so do we really need it?